<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ciudad</class>
 <widget class="QWidget" name="Ciudad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estados</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ciudades:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonGuardar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBorrar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ciudades</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ciudad_text">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Ciudad:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>521</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>Id:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LimpiarCampos">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar Campos</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_estados">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Estados:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
